### PR TITLE
Update mobile_phone.json

### PR DESCRIPTION
**Pull request by giorgiomec**
TIM's company name is TIM Group (officially TIM S.p.A.); Wind is now Wind Tre

### CODE REVIEW

File: data/brands/shop/mobile_phone.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   },
   "items": [
     {
-      "displayName": "3 Store",
+      "displayName": "3",
       "id": "3store-e135a8",
       "locationSet": {"include": ["gb", "it"]},
       "matchNames": ["3", "three", "three store"],
```
Comment (arch0345) [inline data/brands/shop/mobile_phone.json:16]: To match existing 3 Stores to this modified entry
```suggestion
       "matchNames": ["3 store", "three", "three store"],
```

**Review by arch0345 — CHANGES_REQUESTED** [review]

**Comment by UKChris-osm** [comment]
I think three stores are now branded as "_three._" (as in the word, not the number) aren't they?

**Comment by arch0345** [comment]
> I think three stores are now branded as "_three._" (as in the word, not the number) aren't they?

Yeah this seems to be the case - most of if not all the locations I saw on https://locator.three.co.uk/ show these stores being signed as "Three."

**Comment by arch0345** [comment]
> TIM's company name is TIM Group (officially TIM S.p.A.)

TIM Group seems to just be the English translation to Gruppo TIM on Wikidata: maybe this could be changed to Gruppo TIM?

File: data/brands/shop/mobile_phone.json [hunk 2]
```diff
@@ -10,15 +10,15 @@
   },
   "items": [
     {
-      "displayName": "3 Store",
+      "displayName": "",
```
Comment (bhousel) [inline data/brands/shop/mobile_phone.json:13]: It needs a `displayName` here.  It should probably be the same as the name "Three."

File: data/brands/shop/mobile_phone.json [hunk 3]
```diff
@@ -843,14 +843,14 @@
...
+      "displayName": "TIM Group",
       "id": "tim-a04a47",
-      "locationSet": {"include": ["br", "it"]},
+      "locationSet": {"include": ["br", "it", "sm"]},
       "tags": {
         "brand": "TIM",
         "brand:wikidata": "Q144617",
         "brand:wikipedia": "en:Gruppo TIM",
-        "name": "TIM",
+        "name": "TIM Group",
```
Comment (bhousel) [inline data/brands/shop/mobile_phone.json:853]: Are these stores actually called "TIM Group", like on signs and stuff?  
The name is more what people commonly refer to the store as, not the official company name.

File: data/brands/shop/mobile_phone.json [hunk 4]
```diff
@@ -981,14 +981,14 @@
...
-      "displayName": "Wind",
+      "displayName": "Wind Tre",
       "id": "wind-c4d7ba",
       "locationSet": {"include": ["it"]},
       "tags": {
-        "brand": "Wind",
-        "brand:wikidata": "Q650467",
-        "brand:wikipedia": "en:Wind (Italy)",
-        "name": "Wind",
+        "brand": "Wind Tre",
```
Comment (bhousel) [inline data/brands/shop/mobile_phone.json:988]: When stores re-brand, we try to match the old name in an `alt_name` tag or `matchNames` property.  

I'm not sure how completely this rebrand has happened in Italy?  Are all stores now signed as "Wind Tre" or is it still a mix?

**Review by bhousel — CHANGES_REQUESTED** [review]
I have some questions about the changes being proposed..

File: data/brands/shop/mobile_phone.json (hunk 2 above)
Comment (arch0345) [reply data/brands/shop/mobile_phone.json]: Yeah, I forgot to replace the displayName

**Comment by UKChris-osm** [comment]
Currently 3 is set to gb and it, but has the Wind Tre merger taken over from 3 in Italy?

I've added 3 Ireland as I think they still refer to their stores as "3 Stores", but should the Italy location be removed from the current 3 store entry?

**Review by bhousel — APPROVED** [review]

**Comment by bhousel** [comment]
Merging as-is..  It's probably fine.

**Comment by boothym** [comment]
If all of the UK's Three stores have changed their name, shouldn't there be a mass edit of the shops in OSM?

**Pull request merged**